selfdrive/car/__init__.py

```diff
@@ -1,10 +1,14 @@
 # functions common among cars
+from collections import namedtuple
+
 from cereal import car
 from common.numpy_fast import clip
 
 # kg of standard extra cargo to count for drive, gas, etc...
 STD_CARGO_KG = 136.
 
+CarInfo = namedtuple('CarInfo', ['name', 'years', 'supported_package'], defaults=('All',))
```

Comment:
let's remove the defaults, I think all of thse should be explicitly set

Reply:
This is only for the supported_package, a lot of cars support all packages/trims

Reply:
I know, I'd like this to be set explicitly